ConcernsCaseWork/ConcernsCaseWork.API.Tests/UseCases/CaseActions/Decisions/AllDecisionUseCasesTests.cs

```diff
@@ -91,7 +92,8 @@ private TypeInfo[] GetTypes()
                 .Where(x =>
                     x.IsClass &&
                     x.Namespace != null
-                    && x.Namespace.StartsWith(typeToFind.Namespace))
+                    && x.Namespace.Equals(typeToFind.Namespace))
```

Comment:
I couldn't get this to work for decision outcome, it doesn't work with async functions without having to change the code to specifically work for the test